src/math/ec_keys.cpp

```diff
@@ -128,7 +162,7 @@ ec_secret create_nonce(ec_secret secret, hash_digest hash)
     {
         V = hmac_sha256_hash(V, K);
 
-        if (verify_private_key(V))
+        if (0 == index--)
```

Comment:
Maybe, but unless you are a Python developer, the `++` and `--` operators are a perfectly common, everyday operators. I think we can assume that people reading this code are fairly familiar with the language. Otherwise, I suppose we could do this:

    if (0 == index)
        return V;
    --index;